integrations/adding-an-integration/conneceting-to-k8s-cluster.md

```diff
@@ -0,0 +1,23 @@
+# Connecting to K8s cluster
```

Comment:
Nit: Let's title this "Connecting to a Kubernetes cluster" just for clarity's sake.